robot_sdk/wrappers/ec_wrapper.py

```
@@ -67,43 +77,45 @@ def get_joint_angles(self) -> Optional[List[float]]:
-    def move_to_joint_angles(self, joint_angles: List[float]) -> Optional[bool]:
+    def move_to_joint(self, joint_angles: List[float], joint_vel: float = 60, joint_acc: float = 0.01) -> Optional[tuple]:
         '''
         Move the robot to desired joint angles.
         '''
         try:
-            if not isinstance(joint_angles, list):
-                return False
-            joint_angles_deg = np.rad2deg(joint_angles)
-            return self.client.movej(joint_angles=joint_angles_deg.tolist())
+            _, response = self.client.movej(joint_angles=joint_angles, joint_vel=joint_vel, joint_acc=joint_acc)
+            return _, response
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)